clients/java/src/main/java/io/camunda/zeebe/client/impl/search/filter/UserTaskFilterImpl.java

```diff
@@ -84,7 +84,7 @@ public UserTaskFilter processInstanceKey(final Long processInstanceKey) {
 
   @Override
   public UserTaskFilter tentantId(final String tenantId) {
```

Comment:
this is not related to this PR by `tentantId` typo needs to be fixed

Reply:
I had included on the PR description

" Fix - rename tenantIds to tenantId on Search User Task request/response"

I can either rename the PR title or create a new one for that

But I dont believe is necessary a new PR to apply this fix.